README.md

```diff
@@ -1,2 +1,168 @@
+|minimumDailyDose|double|e.g. 6.25|Unit: mg/day. May only occur once.|
+|targetDailyDose|double|e.g. 50.0|Unit: mg/day. May only occur once.|
+
+Q: Do we possibly need a connection from a single medication to a video?
+
+### /medicationClasses/$medicationClassId$
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|id|string|e.g. "BETABLOCKER"||
+|name|LocalizedText|-|A name for a given medicationClass to be displayed to a user.|
+|videoSectionId|string|-|To be used to retrieve a video using /videos/$videoSectionId$/videos/$videoId$.|
+|videoId|string|-|To be used to retrieve a video using /videos/$videoSectionId$/videos/$videoId$.|
+
+Q: Do we necessarily need a connection between medication classes and videos / videoSections?
+
+## /videoSections
+
+In this section, we describe all data related to educational videos to be shown in the Engage-HF mobile apps. The videos are grouped into different categories to be displayed as sections in the mobile apps.
+
+### /videoSections/$videoSectionId$
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|title|LocalizedText|e.g. "ENGAGE-HF Application"|May be localized.|
+|description|LocalizedText|e.g. "Helpful videos on the ENGAGE-HF mobile application."|May be localized. Are there different videos / videoSections for each platform?|
+
+Q: Can we specify a certain order in Firestore or should we add something like an id to order it by?
+Q: All videos are shown to all users, or is there any filtering/reordering/difference for any users with respect to other users?
+Q: Would we gain a benefit from linking the relevant videos / videoSections to the medications / medication classes?
+
+### /videoSections/$videoSectionId$/videos/$videoId$
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|title|LocalizedText|e.g. "Beta Blockers for Heart Failure"|May be localized.|
+|url|LocalizedText|e.g. "https://youtu.be/XfgcXkq61k0"|May be localized.|
+
+Q: Can we specify a certain order in Firestore or should we add something like an id to order it by?
+Q: All videos are shown to all users, or is there any filtering/reordering/difference for any users with respect to other users?
+
+## /users/$userId$
+
+In this section, we describe all user-related data to be stored.
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|fullName|string|-|Contains a user's full name. TBD: May potentially be split up into firstName/lastName or firstName/middleNames/familyName.|
+|dateOfBirth|Date|-|To be used for verification purposes.|
+|invitationCode|string|-|The invitationCode to be used when logging in to the app for the first time.|
+|language|optional string|e.g. "en"|Following IETF BCP-47 / [FHIR ValueSet languages](https://hl7.org/fhir/R4B/valueset-languages.html)|
+
+Q: Can we somehow ensure that a given logged in user can only access their own information? Clinicians require access to more patients, but they are currently not reflected as "users".
```

Comment:
We have firebase security rules that we use to enforce this. We have already setup them up [here](https://github.com/StanfordBDHG/ENGAGE-HF-iOS/blob/main/firestore.rules) and we should pull them in this repo. We need to adapt this rule set for all other elements here as well.

Reply:
Sounds good!